.github/workflows/meshtasticd-integration.yml

```
@@ -34,6 +40,99 @@ jobs:
     steps:
       - uses: actions/checkout@v6
 
+      - name: Derive meshtasticd image cache metadata
+        id: meshtasticd-cache
+        shell: bash
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2129:style:8:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)